### PR TITLE
Ask for school district name for teacher application custom schools

### PR DESCRIPTION
**Pull request by aoby**
image

### CODE REVIEW

**Comment by aoby** [comment]
[If someone doesn't find their school in the school dropdown, also ask for school district in follow up fields](https://trello.com/c/UBJcB9RF/505-if-someone-doesnt-find-their-school-in-the-school-dropdown-also-ask-for-school-district-in-follow-up-fields)

File: dashboard/test/ui/features/pd/teacher_application.feature [hunk 1]
```diff
@@ -1,13 +1,13 @@
 @dashboard_db_access
-@eyes
+#@eyes
```
Comment (clareconstantine) [inline dashboard/test/ui/features/pd/teacher_application.feature]: was that `#` intentional?

File: dashboard/test/ui/features/pd/teacher_application.feature [hunk 2]
```diff
@@ -98,7 +99,7 @@ Scenario: Basic teacher application submission
 
   # Confirmation page
   Then I wait until element "h1" contains text "Thank you for submitting your application to join Code.org’s Professional Learning Program!"
-  Then I see no difference for "Confirmation"
+#  Then I see no difference for "Confirmation"
```
Comment (clareconstantine) [inline dashboard/test/ui/features/pd/teacher_application.feature]: did you mean to comment this out?

File: dashboard/test/ui/features/pd/teacher_application.feature (hunk 1 above)
Comment (aoby) [reply dashboard/test/ui/features/pd/teacher_application.feature]: No, thanks for catching

File: dashboard/test/ui/features/pd/teacher_application.feature (hunk 2 above)
Comment (aoby) [reply dashboard/test/ui/features/pd/teacher_application.feature]: No, same as above. I commented out the eyes parts to test more rapidly on localhost, and clearly forgot to add them back in. Thanks for catching!

**Pull request merged**